p4_pdpi/entity_management.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
```

Comment:
I thought these don't work and can't be open-sourced right now?

Reply:
True, but for simplicity I only removed the BUILD targets and kept the source files.